### PR TITLE
Add security policy.

### PR DESCRIPTION
**Pull request by MikeNeilson**
Was going through various settings and realized we haven't setup a security policy document yet.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+| 7.0.12   | :white_check_mark: |
+| > 6.8.14 | :white_check_mark: |
+
+8.0 is in development and if you are attempting to use that we welcome reports but responses will likely be delayed.
+
+## Reporting a Vulnerability
+
+To report a vulnerability please browses to the issues, click New Issues, and select Report a Vulnerability.
+
+Provided as much delete as you are willing so we can get started on fixing any issues.
```
Comment (ktarbet) [inline SECURITY.md:18]:
```suggestion
Provide as much delete as you are willing so we can get started on fixing any issues.
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+| Version  | Supported          |
+| -------  | ------------------ |
+| 7.0.12   | :white_check_mark: |
+| > 6.8.14 | :white_check_mark: |
+
+8.0 is in development and if you are attempting to use that we welcome reports but responses will likely be delayed.
+
+## Reporting a Vulnerability
+
+To report a vulnerability please browses to the issues, click New Issues, and select Report a Vulnerability.
```
Comment (ktarbet) [inline SECURITY.md:16]:
```suggestion
To report a vulnerability please go to [issues](https://github.com/opendcs/opendcs/issues), click New Issues, and select Report a Vulnerability.
```

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+| 7.0.12   | :white_check_mark: |
+| > 6.8.14 | :white_check_mark: |
+
+8.0 is in development and if you are attempting to use that we welcome reports but responses will likely be delayed.
+
+## Reporting a Vulnerability
+
+To report a vulnerability please go to [issues](https://github.com/opendcs/opendcs/issues), click New Issues, and select Report a Vulnerability.
+
+Provide as much delete as you are willing so we can get started on fixing any issues.
```
Comment (adamkorynta) [inline SECURITY.md:18]: typo `delete` -> `detail`

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+
+## Regarding Static Analysis tools
+
+The OpenDCS project, and it's sibling projects, use a combination of SonarCloud, Spotbugs, PMD, and checkstyle to
```
Comment (adamkorynta) [inline SECURITY.md:23]: can add Dependabot to analysis tools

**Review by adamkorynta — APPROVED** [review]

**Pull request merged**